content/fr/docs/setup/pick-right-solution.md

```diff
@@ -0,0 +1,302 @@
+
+* [Amazon Elastic Container Service for Kubernetes](https://aws.amazon.com/eks/) offre un service managé de Kubernetes.
+
+* [Azure Kubernetes Service](https://azure.microsoft.com/services/container-service/) offre des clusters Kubernetes managés.
+
+* [Containership Kubernetes Engine (CKE)](https://containership.io/containership-platform) Approvisionnement et gestion intuitive de clusters
+ Kubernetes sur GCP, Azure, AWS, Packet, et DigitalOcean. Mises à niveau transparentes, mise à l'échelle automatique, métriques, création de 
+workloads, et plus encore.
+
+* [DigitalOcean Kubernetes](https://www.digitalocean.com/products/kubernetes/) offre un service managé de Kubernetes.
+
+* [Giant Swarm](https://giantswarm.io/product/) offre des clusters Kubernetes managés dans leur propre centre de données, on-premises ou sur des clouds public.
+
+* [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) offre des clusters Kubernetes managés.
+
+* [IBM Cloud Kubernetes Service](https://console.bluemix.net/docs/containers/container_index.html) offre des clusters Kubernetes managés
+ avec choix d'isolation, des outils opérationnels, une vision intégrée de la sécurité des images et des conteneurs et une intégration avec Watson, IoT et les données.
+
+* [Kubermatic](https://www.loodse.com) fournit des clusters Kubernetes managés pour divers clouds publics, y compris AWS et Digital Ocean, ainsi que sur site avec intégration OpenStack.
+
+* [Kublr](https://kublr.com) offre des clusters Kubernetes sécurisés, évolutifs et hautement fiables sur AWS, Azure, GCP et on-premises,
+ de qualité professionnelle. Il inclut la sauvegarde et la reprise après sinistre prêtes à l'emploi, la journalisation et la surveillance centralisées multi-clusters, ainsi qu'une fonction d'alerte intégrée.
+
+* [Madcore.Ai](https://madcore.ai) est un outil CLI orienté développement pour déployer l'infrastructure Kubernetes dans AWS. Masters, autoscaling group nodes avec des spot instances, ingress-ssl-lego, Heapster, et Grafana.
+
+* [Nutanix Karbon](https://www.nutanix.com/products/karbon/) est une plateforme de gestion et d'exploitation Kubernetes multi-clusters et hautement disponible qui simplifie l'approvisionnement, les opérations et la gestion du cycle de vie de Kubernetes.
+
+* [OpenShift Dedicated] (https://www.openshift.com/dedicated/) offre des clusters Kubernetes gérés et optimisés par OpenShift.
+
+* [OpenShift Online](https://www.openshift.com/features/) fournit un accès hébergé gratuit aux applications Kubernetes.
+
+* [Oracle Container Engine for Kubernetes] (https://docs.us-phoenix-1.oraclecloud.com/Content/ContEng/Concepts/contengoverview.htm) est un service entièrement géré, évolutif et hautement disponible que vous pouvez utiliser pour déployer vos applications conteneurisées dans le cloud.
```

Comment:
Lien markdown cassé

Reply:
on https://deploy-preview-12658--kubernetes-io-master-staging.netlify.com/fr/docs/setup/pick-right-solution/ the link is working, not good for you?

Reply:
ok, got it. Syntax checking in IDEs are warning you, btw this good old Hugo works well. 
I fix